### PR TITLE
Edited comments and whitespace, amending #7153.

### PR DESCRIPTION
**Pull request by tschw**

### CODE REVIEW

**Comment by mrdoob** [comment]
Ops! Sorry, do you mind removing `webgl_shadowmap_omnidirectional` from the PR?

**Comment by tschw** [comment]
Do you mind if I just resolve the merge conflict?

File: examples/webgl_shadowmap_pointlight.html [hunk 1]
```diff
@@ -162,16 +161,28 @@
...
+				// wall.name = "front wall";
+				// wall.scale.multiplyScalar( 3 );
+				// wall.castShadow = false;
+				// wall.receiveShadow = true;
+				// scene.add( wall );
+				// wall.position.set( 0, 10, 14 );
+				// wall.rotation.y = Math.PI / 2;
+				// wall.rotation.z = Math.PI / 2;
+
+				var sphereGeometry = new THREE.SphereGeometry( 1, 32, 32 );
```
Comment (mrdoob) [inline examples/webgl_shadowmap_pointlight.html]: This is not needed.

File: examples/webgl_shadowmap_pointlight.html [hunk 2]
```diff
@@ -162,16 +161,28 @@
...
+				var material = new THREE.MeshBasicMaterial( { color: 0xffffff } );
+				var sphere = new THREE.Mesh( sphereGeometry, material );
+				sphere.castShadow = false;
+				sphere.receiveShadow = false;
+				sphere.position.set( 0, 11, 4 );
+				scene.add( sphere );
+
+				pointLightParent = new THREE.Object3D();
```
Comment (mrdoob) [inline examples/webgl_shadowmap_pointlight.html]: This is not needed either.

**Comment by tschw** [comment]
OK, changes removed.

**Pull request merged**